include/boost/beast/experimental/core/uri/buffer.hpp

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2018 oneiric (oneiric at i2pmail dot org)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/boostorg/beast
+//
+
+#ifndef BOOST_BEAST_CORE_URI_BUFFER_HPP
+#define BOOST_BEAST_CORE_URI_BUFFER_HPP
+
+#include <vector>
```

Comment:
standard library includes should come last

Reply:
Ok, some projects like them first, will move it.